graftm/pplacer.py

```diff
@@ -40,39 +40,49 @@ def alignment_merger(self, alignment_files, output_alignment_path):
                 for record in alignments: # For each record in the read list
                     record.id = record.id + '_' + str(file_number) # append the unique identifier to the record id
                 SeqIO.write(alignments, output, "fasta") # And write the reads to the file
-                alias_hash[str(file_number)] = {'output_path': os.path.join(os.path.dirname(alignment_file),'placements.jplace'),
+                alias_hash[str(file_number)] = {'output_path': os.path.join(os.path.dirname(alignment_file), 'placements.jplace'),
                                                 'place': []}
                 file_number += 1
         return alias_hash
 
-    def jplace_split(self, jplace_file, alias_hash):
+    def jplace_split(self, original_jplace, alias_hash):
```

Comment:
If you like. It has the same changes, so you havent lost ground. I made an extra branch because I was fixing another, unrelated issue and needed the changes in this branch.